config/Makefile

```diff
@@ -1,16 +1,24 @@
+# this is a region used for materializing configuration
+# this one has been chosen because the resulting region short name
+#  * is 5 characters long
+#  * ends with a number
+# and thus name length limits and naming patterns can be tested
+# this is also in alignment with personal dev environments, where the short region name is a 5 character string
+REGION=southwestus
+
 materialize:
 	@echo "Materializing public cloud int env of config/config.yaml"
-	@../templatize.sh dev > public-cloud-dev.json
-	@echo "Materializing public cloud cs-pr env of config/config.yaml"
-	@../templatize.sh cs-pr > public-cloud-cs-pr.json
-	@echo "Materializing nightly env of config/config.yaml"
-	@../templatize.sh nightly > public-cloud-nightly.json
+	@REGION=$(REGION) ../templatize.sh dev > public-cloud-dev.json
+	@echo "Materializing public cloud cspr env of config/config.yaml"
+	@REGION=$(REGION) ../templatize.sh cspr > public-cloud-cspr.json
+	@echo "Materializing nitey env of config/config.yaml"
```

Comment:
```suggestion
	@echo "Materializing nightly env of config/config.yaml"
```

Reply:
(and elsewhere)

Reply:
Seemed like a typo - if we are going for shorter, let's try `ntly` or something

Reply:
thanks for the name suggestion